tests/core/basic20plus/task.yaml

```diff
@@ -32,23 +38,20 @@ execute: |
     snap changes | MATCH "Done.*Initialize system state"
 
     echo "Check that a simple shell snap"
-    if os.query is-core22; then
-        snap install --edge "test-snapd-sh-${base_snap}"
```

Comment:
Do you know why we used `--edge`?

Reply:
This was before it was pushed to stable (I.e) before the core22 base reached stable

Reply:
well, we agreed with store team to keep test snaps in edge and non listed. I think this is the main raeson